README.md

```diff
@@ -29,3 +30,29 @@ You can use a different registry by overwriting the `IMAGE_REGISTRY` variable:
 ```shell
 make -e IMAGE_REGISRY=quay.io/YOUR_USER/notebooks base-ubi8-python-3.8
 ```
+
+### Building jupyter-minimal-ubi8-python-3.8 image
+
+The jupyter-minimal-ubi8-python-3.8 is a lightweight image for running the Jupyter notebook server.
+
+We use the base-ubi8-python-3.8 image [quay.io/opendatahub/notebooks:base-ubi8-python-3.8](https://quay.io/repository/opendatahub/notebooks?tab=tags) as a builder image.
+
```

Comment:
The `base-ubi8-python-3.8` is actually the base image, not the builder image. We are not using this image to build jupyter.

s/as a builder image/as the base image